### ISSUE TITLE
create test_app rake task that will bootstrap the dummy application

### ISSUE THREAD

**Issue by peterberkenbosch** [denkGroot/Spina#128]
Like on Spree and Solidus, so we can dynamically bootstrap a rails sample app for testing. This reflect the real world better then static configured one.

**Comment by Bramjetten** [comment]
I think rails g spina:install already does this when running from the dummy directory. Or should we create a task to really build the dummy app from scratch?

**Comment by peterberkenbosch** [comment]
yeah, something like this https://github.com/solidusio/solidus/blob/master/core/Rakefile 

I can take care of that, if needed.

**Comment by dankmitchell** [comment]
@peterberkenbosch still keen to help out? 👀
